linux/deploy_vm/ubuntu/prepare_ubuntu_iso_install.yml

```diff
@@ -14,34 +14,19 @@
       {%- else -%}unknown
       {%- endif -%}
 
-- name: "Set fact of Ubuntu Server"
-  block:
-    - name: "Set Ubuntu autoinstall method"
-      ansible.builtin.set_fact:
-        ubuntu_install_method: |-
-          {%- if ubuntu_version and ubuntu_version is version('20.04', '<') -%}simulation
-          {%- else -%}cloud-init
-          {%- endif -%}
-
-    - name: "Set fact of Ubuntu unattend install file"
-      ansible.builtin.set_fact:
-        unattend_install_conf: "Ubuntu/Server/user-data.j2"
-      when: ubuntu_install_method == "cloud-init"
+- name: "Set fact of Ubuntu Server unattend install file"
+  ansible.builtin.set_fact:
+    unattend_install_conf: "Ubuntu/Server/user-data.j2"
   when: ubuntu_edition == 'server'
 
 - name: "Set fact of Ubuntu desktop"
   block:
-    - name: “Set autoinstall method for Ubuntu Desktop”
+    - name: “Set fact of unattend install file for Ubuntu Desktop”
       ansible.builtin.set_fact:
         unattend_install_conf: "Ubuntu/Desktop/Ubiquity/ubuntu.seed"
```

Comment:
Please add a condition here: `when: ubuntu_version and ubuntu_version is version('23.04', '<')`